package.json

```diff
@@ -43,7 +43,7 @@
     "react-content-loader": "^6.2.0",
     "react-datepicker": "^4.8.0",
     "react-dom": "18.1.0",
-    "react-helsinki-headless-cms": "1.0.0-alpha2-canary-bc85c6d",
+    "react-helsinki-headless-cms": "1.0.0-alpha2-canary-6084e6e",
```

Comment:
This might need a new release, so it does not break the changes that are done later.